easybuild/easyconfigs/a/ALPS/ALPS-2.3.0-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'ALPS'
+version = '2.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "http://alps.comp-phys.org/"
+description = "ALPS project"
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+toolchainopts = {'cstd': 'c++03'}
+
+sources = ['%(namelower)s-%(version)s-src.tar.gz']
+source_urls = ['http://alps.comp-phys.org/static/software/releases/']
+
+boostver = '1.63.0'
+pyver = '2.7.12'
+
+dependencies = [
+    ('CMake', '3.6.2'),
+    ('Boost', boostver, '-Python-%s' % pyver),
```

Comment:
@dolfim just use `versionsuffix` here? Also, you don't need `boostver` (anymore), since it's only used in one place?

```python
('Boost', '1.63.0`, versionsuffix),
```